apps/ocpp/call_handlers/errors/certificates.py

```
@@ -0,0 +1,190 @@
+
+from channels.db import database_sync_to_async
+from django.utils import timezone
+
+from ... import store
+from ...models import CertificateOperation, InstalledCertificate
+from ..types import CallErrorContext
+from ..utils import _json_details
+
+
+async def handle_install_certificate_error(
+    consumer: CallErrorContext,
```

Ruff (0.14.14)

[warning] Unused function argument: `consumer`

(ARG001)